tests/version_test.go

```diff
@@ -41,7 +41,7 @@ var _ = Describe("[sig-compute]Version", decorators.SigCompute, func() {
 	})
 
 	Describe("Check that version parameters where loaded by ldflags in build time", func() {
-		It("[test_id:555]Should return a good version information struct", func() {
+		It("[test_id:555]Should return a good version information struct", decorators.Conformance, func() {
```

Comment:
This test will not work outside of CI. You can build the test with different tools than the Kubevirt deployed...

Reply:
right! Thanks